src/scanner/present.rs

```diff
@@ -359,3 +383,47 @@ async fn answer_active_oracle(
 
     Ok(())
 }
+
+async fn is_active_oracle_expired(
```

Comment:
Should we change this fn name? Since it also updates the expiration on db?

Reply:
It's probably ok. The plan really is to remove the update once all the db records are updated the first time. Only temporaru